framework/db/oci/Schema.php

```diff
@@ -219,7 +215,15 @@ protected function createColumn($column)
             if (stripos($column['DATA_DEFAULT'], 'timestamp') !== false) {
                 $c->defaultValue = null;
             } else {
-                $c->defaultValue = $c->phpTypecast($column['DATA_DEFAULT']);
+                $defaultValue = $column['DATA_DEFAULT'];
+                if ($defaultValue !== null) {
+                    if (($len=strlen($defaultValue)) > 2 && $defaultValue[0] === "'" && $defaultValue[$len-1] === "'") {
+                        $defaultValue = substr($column['DATA_DEFAULT'], 1, -1);
```

Comment:
I was also thinking about trying to detect numerics here and if the default value is neither a string nor numeric it could be wrapper as `db\Expression`.

When processing datatypes this `Schema` removes default values for timestamps. I don't know why, should this be fixed?